packages/reactive_ble_mobile/android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +0,0 @@
-#Wed Sep 29 14:33:41 CEST 2021
-distributionBase=GRADLE_USER_HOME
```

Comment:
Is it correct this file (also) is removed? 🤔

Reply:
Also, should be there to let devs of the package specify which Gradle wrapper version to use.